Assets/MRTK/Services/InputSimulation/InputSimulationService.cs

```diff
@@ -136,6 +136,11 @@ public void ResetHandRight()
         /// </summary>
         public bool SimulateEyePosition { get; set; }
 
+        /// If true then the gaze data provider is always simulated.
+        /// </summary>
+        public bool SimulateGaze { get; set; }
```

Comment:
Hold up...is this ever set to true? It looks like it is only getting set as false in the code.